### PR TITLE
Add internationalisation of dates for worldclock

### PR DESCRIPTION
**Pull request by jordimas**

### CODE REVIEW

File: apps/worldclock/app.js [hunk 1]
```diff
@@ -118,13 +113,12 @@ function draw() {
...
     const dx = getCurrentTimeFromOffset(gmt, offset[OFFSET_HOURS]);
-    hours = doublenum(dx.getHours());
-    minutes = doublenum(dx.getMinutes());
+    var time = require("locale").time(dx, 1);
 
     if (offsets.length === 1) {
       // For a single secondary timezone, draw it bigger and drop time zone to second line
       g.setFont(font, secondaryTimeFontSize);
-      g.drawString(`${hours}:${minutes}`, xyCenter, yposTime2, true);
+      g.drawString(`${time}`, xyCenter, yposTime2, true);
```
Comment (bobrippling) [inline apps/worldclock/app.js:121]: This and below can be like so, to save a bit of processing:
```suggestion
      g.drawString(time, xyCenter, yposTime2, true);
```

**Review by bobrippling — COMMENTED** [review]
One minor comment, no need to fix - happy to merge without, let me know either way

**Comment by jordimas** [comment]
> One minor comment, no need to fix - happy to merge without, let me know either way

Good suggestion, I accepted. Thanks

**Comment by jordimas** [comment]
When merging the PR, considering doing a  squash. Usually people is not interested on the small commits done underdevelopment, they care about the large merge with the new functionality.

**Comment by bobrippling** [comment]
Gotya - I think for 4 commits it's ok but shall bear that in mind more when bringing in branches, and thanks!

**Pull request merged**